README.md

```diff
@@ -40,7 +40,10 @@ Nx.default_backend({EMLX.Backend, device: :cpu})
 Nx.default_backend({EMLX.Backend, device: :gpu})
 ```
 
-If desireable, you can also set the default compiler:
+If you want to use the JIT compiler, you can set the default compiler as shown below.
+
+Currently, the underlying implementation is not totally safe and could lead to a deadlocked dirty NIF, so this must be used with care.
```

Comment:
@cocoa-xu is this still valid?

Reply:
I think we can remove this. It's actually not because of a deadlock, it was the registered tag was unregistered too early. And speaking of this, there needs to be a follow-up PR to manually unregister the tag in `nif_call`. I'll create an issue for this.